rasa/core/actions/flows.py

```diff
@@ -156,10 +157,22 @@ async def run(
         corrected_slots = context.get("corrected_slots", {})
         reset_point = context.get("corrected_reset_point", {})
```

Comment:
What is the reset point in this context? This will help me understand why we break the below loop in L160 when the reset point id matches the flow id.

Reply:
when a user corrects a the answer to a prior question, we jump back in the flow to that question to make sure, we follow the right branches afterwards -> a different answer to that question might lead us down a different path. the reset point is the step we need to jump too. (shouldn'T be called point though, will rename)